--- src/dictionary.cpp	2024-08-14 08:40:19.407432860 +0530
+++ src/dictionary.cpp	2024-08-14 08:40:28.415432860 +0530
@@ -65,7 +65,7 @@
 }
 
 template <typename T1, typename T2>
-struct pair_1st_cmp: public std::binary_function<bool, T1, T2> {
+struct pair_1st_cmp: public std::__binary_function<bool, T1, T2> {
   bool operator()(const std::pair<T1, T2> &x1,
                   const std::pair<T1, T2> &x2)  {
     return x1.first < x2.first;
